Source/DfBAdminToolkit/View/PaperView.designer.cs

```diff
@@ -185,27 +185,62 @@ private void InitializeComponent()
             office2010White1.TextColor = System.Drawing.Color.Black;
             this.buttonEx_PaperLoadPaper.ColorTable = office2010White1;
             this.buttonEx_PaperLoadPaper.Dock = System.Windows.Forms.DockStyle.Bottom;
-            this.buttonEx_PaperLoadPaper.Location = new System.Drawing.Point(244, 6);
-            this.buttonEx_PaperLoadPaper.Margin = new System.Windows.Forms.Padding(4, 5, 4, 5);
+            this.buttonEx_PaperLoadPaper.Location = new System.Drawing.Point(256, 4);
             this.buttonEx_PaperLoadPaper.Name = "buttonEx_PaperLoadPaper";
-            this.buttonEx_PaperLoadPaper.Size = new System.Drawing.Size(220, 37);
+            this.buttonEx_PaperLoadPaper.Size = new System.Drawing.Size(234, 24);
             this.buttonEx_PaperLoadPaper.TabIndex = 4;
             this.buttonEx_PaperLoadPaper.Text = "Load Paper Docs";
             this.buttonEx_PaperLoadPaper.Theme = DfBAdminToolkit.Common.Component.Theme.MSOffice2010_White;
             this.buttonEx_PaperLoadPaper.UseVisualStyleBackColor = true;
             // 
+            // flowLayoutPanel1
```

Comment:
Relevant new additions start here.  The rest of the changes were made automatically and I believe have no effect as the size of the component is determined by the window size.